yarn.lock

```diff
@@ -3276,6 +3276,11 @@ json5@^0.5.0:
   version "0.5.0"
   resolved "https://registry.yarnpkg.com/json5/-/json5-0.5.0.tgz#9b20715b026cbe3778fd769edccd822d8332a5b2"
 
+jsonabc@^2.3.1:
+  version "2.3.1"
+  resolved "https://registry.yarnpkg.com/jsonabc/-/jsonabc-2.3.1.tgz#800a1bd158fb30ace2b4613f8725adfcb040bfb4"
+  integrity sha1-gAob0Vj7MKzitGE/hyWt/LBAv7Q=
+
```

Comment:
You should run `yarn install` again, jsonabc isn't needed anymore

Reply:
@amcgee Thanks for catching that!  I've made another [PR ](https://github.com/myanghua/datastore-app/commit/3447b3dfdcca50d3aad853d6b8a81f903fd0c146)to address these two changes.